point

```diff
@@ -87,17 +81,38 @@ program
 // program.option('--rpc <method> [params]', 'send a command to the daemon'); // todo
 program.parse(process.argv);
 
-// todo: mix other param arguments into config, like here:
-// let argv;
-// let config = rc('kadence', options(program.datadir), argv);
+const ctx = {};
 
-if (!program.config) {
-    program.config = path.join(program.datadir, 'config.json');
+if (program.datadir) {
+    process.env.DATADIR = program.datadir;
 }
```

Comment:
not sure what happens here if program.datadir is not defined.

Also we can use:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment

Reply:
Nullish assignment is not applicable here, and the app will work fine if the `program.datadir` is not defined, since the default value from the `config/*.yaml` will be used.